src/recipients/RecipientItem.tsx

```diff
@@ -33,13 +32,6 @@ function RecipientItem({ recipient, onSelectRecipient, loading }: Props) {
           </Text>
           {recipient.name && <Text style={styles.phone}>{getDisplayDetail(recipient)}</Text>}
         </View>
-        <View style={styles.rightIconContainer}>
-          {loading ? (
```

Comment:
I think we still need the loading icon here. We want to show it when looking up whether a phone number is connected to an address. Can you confirm this by typing a phone number on the search box and tapping it?

Reply:
@satish-ravi are you suggesting just using the loading prop to show a spinner that turns into a blank space once the lookup is complete? why do we need that?

Reply:
oh ic, a bottom sheet comes up once the lookup completes successfully

Reply:
and an error banner appears if it fails

Reply:
yeah, I think it also navigates to an invite screen if its a phone number not connected to Valora